src/main/resources/templates/addNewProductForm.html

```diff
@@ -26,4 +26,7 @@ <h1>Form</h1>
     <p><input type="submit" value="Submit" /> <input type="reset" value="Reset" /></p>
 </form>
 </body>
+<form th:action="@{/logout}" method="post">
```

Comment:
Is /logout auto-generated?

Reply:
yes /logout auto-generated and redirects back to /login?logout